### DISCUSSION TITLE
set custom icon to listview in powermenu script

### DISCUSSION THREAD

**Discussion by joni84** [davatorium/rofi#1597] [Theming]
basically, what I'm trying to replicate is the new Android 12 powermenu.
I wasn't able to find an option to do this.

this is how I want it to look like:
icon
label

**Comment by DaveDavenport** [comment]
do you have a screenshot?
you want to do this? 
![img](https://raw.githubusercontent.com/davatorium/rofi/next/releasenotes/1.6.0/icons.png)

icon on top, text at bottom?

**Reply by joni84** [reply]
![IMG_20220217_155956](https://user-images.githubusercontent.com/98685516/154508654-68dff22c-07b7-4bfd-839d-04cf354739ee.png)
no, icon font on top and text on bottom

if this isn't possible, then can I assign a custom action to a button? not a keybind, a command

**Reply by DaveDavenport** [reply]
No we don't support font for  icons.  (See https://github.com/davatorium/rofi/issues/1132)
If you have an icon theme with those icons, you can reproduce most of the above view.
